lib/types/coreNote.js

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Tozny e3db
+ *
+ * LICENSE
+ *
+ * Tozny dual licenses this product. For commercial use, please contact
+ * info@tozny.com. For non-commercial use, this license permits use of the
+ * software only by government agencies, schools, universities, non-profit
+ * organizations or individuals on projects that do not receive external
+ * funding other than government research grants and contracts. Any other use
+ * requires a commercial license. For the full license, please see LICENSE.md,
+ * in this source repository.
+ *
+ * @copyright Copyright (c) 2018-19 Tozny, LLC (https://tozny.com)
+ */
+
+'use strict'
+
+import Serializable from './serializable'
+import { write } from 'fs'
```

Comment:
not needed, this fs import should be removed.

Reply:
👍 linter doing some weird auto imports here